src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -93,6 +93,7 @@ type DbConfig struct {
 	FeedType           string                         `json:"feed_type,omitempty"`            // Feed type - "DCP" or "TAP"; defaults based on Couchbase server version
 	AllowEmptyPassword bool                           `json:"allow_empty_password,omitempty"` // Allow empty passwords?  Defaults to false
 	CacheConfig        *CacheConfig                   `json:"cache,omitempty"`                // Cache settings
+	MaxRevsCacheSize   *uint32                        `json:"max_revs_cache_size,omitempty"`  // Maximum number of revisions to store in the revision cache
```

Comment:
Might be personal preference, but dropping the 'max' and leaving as rev_cache_size seems like a cleaner name.